src/dotdom.js

```diff
@@ -37,7 +37,7 @@ module.exports = window;
    * @param {Array} [children] - The child VNode elements
    * @returns {VNode} Returns a virtual DOM instance
    */
-  let createElement = (element, props={}, ...children) => (
+  let c = (element, props={}, ...children) => (
```

Comment:
We are OK with long names here, since they will be minified. So let's keep them long and expressive 😄